playbooks/tasks/portal-health-check-disable.yml

```diff
@@ -12,7 +12,9 @@
   register: result
 
 - name: Stop portal healh check to take portal out of load balancer
-  command: 'docker exec health-check cli/disable "{{ out_of_lb_message }}"'
+  vars:
+    disable_message: "{{  out_of_lb_message | default('') }}"
+  command: 'docker exec health-check ./cli disable "{{ disable_message }}"'
   become: True
   become_user: "user"
```

Comment:
This is not needed, `user` user should be added to `docker` group on all hosts - should be part of a setup.